config.dev.json

```diff
@@ -235,6 +235,34 @@
           "decimals": 0
         }
       ]
+    },
+    {
+      "id": 5,
+      "name": "substrate-devnet-phala",
+      "type": "substrate",
+      "bridge": "",
+      "handlers": [],
+      "nativeTokenSymbol": "pha",
+      "nativeTokenFullName": "pha",
+      "nativeTokenDecimals": 18,
+      "blockConfirmations": 2,
+      "startBlock": 0,
+      "resources": [
+        {
+          "resourceId": "0x0000000000000000000000000000000000000000000000000000000000000001",
+          "type": "erc20",
+          "address": "",
+          "symbol": "PHA",
+          "decimals": 18
```

Comment:
Are the decimals right?

Reply:
this is the erc20 token decimal, so should be 18, on devnet, there is no official PHA token but I think that we could use is a Lock/Release erc20 token just pretend to be PHA which has 18 decimals